testapps/test_pillowtop/tests/utils.py

```diff
@@ -0,0 +1,13 @@
+import functools
+from elasticsearch import exceptions as elasticsearch_exceptions
+from requests import exceptions as requests_exceptions
+
+
+def require_elasticsearch(fn):
```

Comment:
i suspect i may end up deleting this soon - planning on setting up ES-specific tests